PROJECT

```diff
@@ -18,6 +18,6 @@ resources:
   domain: managed.openshift.io
   group: hypershiftlogforwarder.managed.openshift.io
```

Comment:
```suggestion
  group: managed.openshift.io
```
Everywhere else you're using the group "managed.openshift.io". If you intended to create a new subgroup, then you'll need to update all the other group name referenes.